pkg/flare/archive.go

```diff
@@ -261,6 +264,12 @@ func getSystemProbeTelemetry() ([]byte, error) {
 	return getHTTPData(sysProbeClient, url)
 }
 
+func getSystemProbeConfig() ([]byte, error) {
```

Comment:
Could you add a comment saying this is a temporary fix until we align configuration between all containers ?

Reply:
Sorry missed the comment

Why is it temporary?
The system-probe container contains ENV variables that affect the runtime configuration